flutter_mobx/example/pubspec.yaml

```diff
@@ -20,10 +20,18 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.2
+  mobx: ^0.0.19
```

Comment:
use the incremented 0.0.20 version

Reply:
worked after running clean on the `build_runner`

` flutter packages pub run build_runner clean`